pkg/resources/session_parameter.go

```diff
@@ -56,6 +58,8 @@ func CreateSessionParameter(d *schema.ResourceData, meta interface{}) error {
 	db := meta.(*sql.DB)
 	key := d.Get("key").(string)
 	value := d.Get("value").(string)
+	client := sdk.NewClientFromDB(db)
+	ctx := context.Background()
 
 	parameterDefault := snowflake.GetParameterDefaults(snowflake.ParameterTypeSession)[key]
```

Comment:
Can we get rid of this dependency on the snowflake package?

Reply:
done